### PR TITLE
Use "go generate" to auto build the mocks

### PR DESCRIPTION
**Pull request by AlCutter**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ install:
   - go get -d -v -t ./...
 
 script:
+  - go generate github.com/google/trillian/...
```
Comment (pphaneuf) [inline .travis.yml]: Drive-by question: is this the sort of thing that could be used to run protoc?

Comment (AlCutter) [inline .travis.yml]: Yes, and that's coming up next :)

Comment (pphaneuf) [inline .travis.yml]: Nice!

**Comment by Martin2112** [comment]
LGTM if Travis is happy.
